tests/acceptance/17_packages/10_new/unsafe/the_great_package_test.cf

```diff
@@ -40,9 +40,13 @@ bundle agent init
       "test_soft_fail" string => "rhel_8",
```

Comment:
I think that the tests do not support listing multiple failure things .... ie test_skip_needs_work, test_skip_unsupported and test_soft_fail. maybe comment out the other two and move those expressions into test_soft-fail

Reply:
I think this line was added by a different PR